### ISSUE TITLE
Rule proposal: prefer `prop` over `prop={true}`

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#11]
On our team when setting props to `true`, we prefer leaving out the `={true}`. It would be nice to have a rule that enforces this (and I suppose one that can enforce the opposite.

Good:

```jsx
<MyComponent myBooleanProp anotherProp="value" />
```

Bad:

```jsx
<MyComponent myBooleanProp={true} anotherProp="value" />
```
